app/views/hyrax/base/_browse_everything.html.erb

```diff
@@ -3,4 +3,4 @@
 </div>
 <%= button_tag(t('hyrax.upload.browse_everything.browse_files_button'), type: 'button', class: 'btn btn-lg btn-success', id: "browse-btn",
   'data-toggle' => 'browse-everything', 'data-route' => browse_everything_engine.root_path,
-  'data-target' => "##{f.object.persisted? ? 'edit' : 'new'}_#{f.object.model.model_name.param_key}" ) %>
+  'data-target' => "##{f.object.persisted? ? 'edit' : 'new'}_#{f.object.model.model_name.param_key}#{f.object.persisted? ? '_'+f.object.model.id : ''}" ) %>
```

Comment:
@jcoyne Yeah, that's cleaner.  I've made the change.  Please re-review when you get a chance.